Sources/StreamChatUI/Utils/ViewPaginationHandling/ScrollViewPaginationHandler.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A component responsible to handle when to load new pages in a scrollView.
+class ScrollViewPaginationHandler: ViewPaginationHandling {
```

Comment:
Those seem like they can be `final` as no one is really subclassing them, right?

Reply:
Done 👍